packages/http-client-js/src/components/external-packages/ts-http-runtime.ts

```diff
@@ -21,6 +21,7 @@ export const httpRuntimeTemplateLib = createPackage({
         "PipelineRequest",
         "HttpResponse",
         "RawHttpHeaders",
+        "RestError",
```

Comment:
interesting, why we have this diff, anything ts emitter missed?

Reply:
no, it is not imported for client emitter but needed in the mcp emitter.